src/app/calendar/ParcelCalendar.tsx

```diff
@@ -13,20 +23,67 @@ const Centerer = styled.div`
 
 const CalendarWrapper = styled.div`
     width: 100vmin;
-    margin: 2em;
+    margin: 0 2em 2em;
 `;
 
-interface ParcelCalendarProps {
-    events: CalendarEvent[];
-}
-
 const ParcelCalendar: React.FC<ParcelCalendarProps> = (props) => {
+    const chosenTheme = useTheme();
+
+    const colorMap: LocationColorMap = {
+        "Brixton Hill - Methodist Church": {
+            color: `${chosenTheme.rainbow.red.background[0]}`,
+            text: `${chosenTheme.rainbow.red.foreground[0]}`,
+        },
+        "Clapham - St Stephens Church": {
+            color: `${chosenTheme.rainbow.orange.background[0]}`,
+            text: `${chosenTheme.rainbow.orange.foreground[0]}`,
+        },
+        "N&B - Emmanuel Church": {
+            color: `${chosenTheme.rainbow.yellow.background[0]}`,
+            text: `${chosenTheme.rainbow.yellow.foreground[0]}`,
+        },
+        "Streatham - Immanuel & St Andrew": {
+            color: `${chosenTheme.rainbow.green.background[0]}`,
+            text: `${chosenTheme.rainbow.green.foreground[0]}`,
+        },
+        "Vauxhall Hope Church": {
+            color: `${chosenTheme.rainbow.green.background[1]}`,
+            text: `${chosenTheme.rainbow.green.foreground[1]}`,
```

Comment:
No 0 and 1 are not always used together 
0 essentially means light and 1 means dark

In general the whole color palette uses indexes to access color gradient in an array

I could have made lightGreen, darkerGreen instead of green[0] and green[1], but then it would be inconsistent as main and primary color uses indexes to access different gradients

Reply:
I think @DanWilkins2107 and @SamYap-Softwire are right and it would be better to have a lightGreen and darkGreen here instead of an array.